web_transport/sdk/api/owt/quic/web_transport_client_interface.h

```diff
@@ -4,17 +4,17 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef OWT_QUIC_QUIC_TRANSPORT_CLIENT_INTERFACE_H_
-#define OWT_QUIC_QUIC_TRANSPORT_CLIENT_INTERFACE_H_
+#ifndef OWT_WEB_TRANSPORT_WEB_TRANSPORT_CLIENT_INTERFACE_H_
+#define OWT_WEB_TRANSPORT_WEB_TRANSPORT_CLIENT_INTERFACE_H_
 
 #include "owt/quic/export.h"
-#include "owt/quic/quic_definitions.h"
-#include "owt/quic/quic_transport_session_interface.h"
+#include "owt/quic/web_transport_definitions.h"
+#include "owt/quic/web_transport_session_interface.h"
 
 namespace owt {
 namespace quic {
 // A client manages a connection to a QuicTransportServer.
-class OWT_EXPORT QuicTransportClientInterface {
+class OWT_EXPORT WebTransportClientInterface {
  public:
   // https://wicg.github.io/web-transport/#dom-quictransportconfiguration-server_certificate_fingerprints.
   struct OWT_EXPORT Parameters {
```

Comment:
Remove the OWT_EXPORT here as it's public and WebTransportClientInterface already prefixed with OWT_EXPORT

Reply:
Fixed.